inspirehep/modules/records/receivers.py

```diff
@@ -148,7 +153,7 @@ def enhance_after_index(sender, json, *args, **kwargs):
 
 
 def populate_bookautocomplete(sender, json, *args, **kwargs):
```

Comment:
so what's the difference between `*autocomplete` and `*_suggest`? if there is none, the names should probably made consistent one way or another.

Reply:
@michamos we will stick to the `*_suggest` as this is the proper way of implementing the suggesters.